scripts/stop_container.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+# Stop the running container (if any)
+containerid=$(docker ps -q)
+if [ -n "$containerid" ]; then
+  docker stop $containerid && docker rm -f $containerid
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)